packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -219,6 +219,90 @@ module.exports = {
               },
             ],
           },
+          // "SCSS loader will create general stylesheets, not including CSS Modules"
```

Comment:
The `scss` loaders needed to precede the `file-loader`, otherwise the Sass modules wouldn't load.